docs/getting_started.md

```diff
@@ -1,219 +1,176 @@
 
-![build](./images/multi-stage-aci.png)
+Once the pipeline is finished, check the execution result:
 
-and check out the published training pipeline in the **mlops-AML-WS** workspace in [Azure Portal](https://portal.azure.com/):
+![Build](./images/multi-stage-aci.png)
 
-![training pipeline](./images/training-pipeline.png)
+Also check the published training pipeline in the **mlops-AML-WS** workspace in [Azure Portal](https://portal.azure.com/):
 
-Great, you now have the build pipeline set up which automatically triggers every time there's a change in the master branch.
+![Training pipeline](./images/training-pipeline.png)
 
-* The first stage of the pipeline, **Model CI**, performs linting, unit testing, build and publishes an **ML Training Pipeline** in an **ML Workspace**.
+Great, you now have the build pipeline set up which automatically triggers every time there's a change in the master branch!
 
-  **Note:** The build pipeline also supports building and publishing ML
-pipelines using R to train a model. This is enabled
-by changing the `build-train-script` pipeline variable to either of:
-* `diabetes_regression_build_train_pipeline_with_r.py` to train a model
-with R on Azure ML Compute. You will also need to uncomment (i.e. include) the
-`r-essentials` Conda packages in the environment definition
-`diabetes_regression/conda_dependencies.yml`.
-* `diabetes_regression_build_train_pipeline_with_r_on_dbricks.py`
-to train a model with R on Databricks. You will need
-to manually create a Databricks cluster and attach it to the ML Workspace as a
-compute (Values DB_CLUSTER_ID and DATABRICKS_COMPUTE_NAME variables should be
-specified). Example ML pipelines using R have a single step to train a model. They don't demonstrate how to evaluate and register a model. The evaluation and registering techniques are shown only in the Python implementation.
+* The first stage of the pipeline, **Model CI**, performs linting, unit testing, code coverage, building and publishes an **ML Training Pipeline** in an **ML Workspace**.
 
 * The second stage of the pipeline, **Train model**, triggers the run of the ML Training Pipeline. The training pipeline will train, evaluate, and register a new model. The actual computation is performed in an [Azure Machine Learning Compute cluster](https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-set-up-training-targets#amlcompute). In Azure DevOps, this stage runs an agentless job that waits for the completion of the Azure ML job, allowing the pipeline to wait for training completion for hours or even days without using agent resources.
 
-**Note:** If the model evaluation determines that the new model does not perform better than the previous one then the new model will not be registered and the pipeline will be cancelled.
+**Note:** If the model evaluation determines that the new model does not perform better than the previous one, the new model will not be registered and the pipeline will be cancelled.
```

Comment:
Please add the step which will fail, the error message, and possibly the link in the evaulate code too, so we don't get _another_ issue filed on this topic.